### PR TITLE
Create core project skeleton

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

File: core/build.gradle [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 26
+
+    defaultConfig {
+        minSdkVersion 21
```
Comment (wtrocki) [inline core/build.gradle]: [Idea] If we want to keep this consistent across all sdk modules we can use constants and import them.
For example here:
https://github.com/facebook/facebook-android-sdk/blob/master/constants.gradle

File: core/src/androidTest/java/org/aerogear/android/core/ExampleInstrumentedTest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package org.aerogear.android.core;
```
Comment (wtrocki) [inline core/src/androidTest/java/org/aerogear/android/core/ExampleInstrumentedTest.java]: This can be removed.

**Comment by wtrocki** [comment]
@danielpassos Build locally. It will be good to have some basic readme update

**Comment by danielpassos** [comment]
@wtrocki Which kind of info you wanna have in the README since we have nothing yet?

**Comment by wtrocki** [comment]
I was just suggesting to add something small. Its good practice for community to just have one sentence what is the project doing and that it is in development. Empty Readme gives zero info. Aerogear has now couple sdk repos so this will reduce confusion.

**Comment by danielpassos** [comment]
@wtrocki I got your point here but still having no idea what to add there. Feel free to send a PR with your thoughts

**Comment by danielpassos** [comment]
@secondsun Wanna double check before I merge it?

**Comment by wtrocki** [comment]
@danielpassos Ignore this comment. Feel free to merge.

**Pull request merged**